packages/bpk-component-breakpoint/src/BpkBreakpoint-test.tsx

```diff
@@ -25,7 +25,7 @@ describe('BpkBreakpoint', () => {
   it('should render if the breakpoint is matched', () => {
     jest.resetModules();
     const BpkBreakpoint = require('./BpkBreakpoint').default; // eslint-disable-line global-require
-    jest.mock('react-responsive', () => (props: any) => props.children(true));
+    jest.mock('./useMediaQuery', () => () => true);
```

Comment:
react-responsive previously called `children(matches)` or in this case `children(true)` to signify that the breakpoint matches, we now do this ourselves [here](https://github.com/Skyscanner/backpack/pull/3284/files#r1526566596). 

Now, we only use an import for finding out if our breakpoint matches, so this is what we're mocking.